initrd/init

```diff
@@ -69,35 +55,76 @@ if [ ! -z "$CONFIG_USB_BOOT_DEV" ]; then
 	echo >> /etc/fstab "$CONFIG_USB_BOOT_DEV /media auto defaults,ro 0 0"
 fi
 
-if [ ! -x "$CONFIG_BOOTSCRIPT" ]; then
-	recovery 'Boot script missing?  Entering recovery shell'
-	# just in case...
-	tpm extend -ix 4 -ic recovery
-	exec /bin/ash
+# Now it is safe to print a banner
+if [ ! -z "$CONFIG_LINUXBOOT" ]; then
+	MOTD=/etc/motd.nerf
+else
+	MOTD=/etc/motd.coreboot
+fi
+if [ -r "$MOTD" ]; then
+	cat "$MOTD"
+	if [ ! -z "$CONFIG_BOOT_RECOVERY_SERIAL" ]; then
+		cat "$MOTD" > "$CONFIG_BOOT_RECOVERY_SERIAL"
+	fi
 fi
 
-# If the user has been holding down r, enter a recovery shell
-# otherwise immediately start the configured boot script.
-# We don't print a prompt, since this is a near instant timeout.
-read \
-	-t 0.1 \
-	-n 1 \
-	boot_option
-echo
-
-if [ "$boot_option" = "r" ]; then
-	# Start an interactive shell
-	recovery 'User requested recovery shell'
-	# just in case...
-	tpm extend -ix 4 -ic recovery
-	exec /bin/ash
+# Setup remote attestation interface
```

Comment:
The network setup should move to a separate setup file, rather than living here in `/init`.

Reply:
Fine by me - I just wanted minimal equal functionality changes to what you had done for linuxboot.